.travis.yml

```diff
@@ -79,10 +75,8 @@ jobs:
         - npm run build
 
     - stage: test
+      <<: *language_go_1_13
```

Comment:
I find it nicer if is `stage` first then `language` (to stay coherent with the others)

Reply:
But then language gets set to Ruby. Travis is very fragile and mysterious when using the merge operator.